tests/playbooks/tests_route_next_hop_ethernet.yml

```diff
@@ -0,0 +1,188 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Test configuring ethernet devices
+  hosts: all
+  vars:
+    type: veth
+    interface0: ethtest0
+    interface1: ethtest1
+
+  tasks:
+    - name: Set type and interface0
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface0 }}"
+    - name: Show interfaces
+      include_tasks: tasks/show_interfaces.yml
+    - name: Manage test interface
+      include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - name: Assert device is present
+      include_tasks: tasks/assert_device_present.yml
+    - name: Set type and interface1
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface1 }}"
+    - name: Show interfaces
+      include_tasks: tasks/show_interfaces.yml
+    - name: Manage test interface
+      include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - name: Assert device is present
+      include_tasks: tasks/assert_device_present.yml
+    - name: Run the test
+      block:
+        - name: Configure the IP addresses
```

Comment:
This does not only configure IP addresses but also a route.

Reply:
I updated.